style/visual.less

```diff
@@ -70,11 +69,13 @@
     }
 
     .data-title {
-        font-family: @regularSecondaryFontFamily;
-        font-size: 10px;
         text-transform: uppercase;
     }
 
+    .data-title.extra-small {
```

Comment:
Don't we use formatting utils any more?

Reply:
Don't think this comment has high priority before new API visual will be approved

Reply:
You removed the import of formatting utils' `CSS` styles.
I'm wondering why you did so?

That's ok if Dual KPI doesn't use Formatting Utils but Formatting Utils is included into TS and JS code.